Transmission.xcodeproj/project.pbxproj

```diff
@@ -596,6 +596,16 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		0836CBE8281E7EF600715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/MainMenu.xib; sourceTree = "<group>"; };
+		0836CBE9281E7EF600715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/PrefsWindow.xib; sourceTree = "<group>"; };
+		0836CBEA281E7EF600715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/Creator.xib; sourceTree = "<group>"; };
+		0836CBEB281E7EF600715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/AddWindow.xib; sourceTree = "<group>"; };
+		0836CBEC281E7EF600715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/AddMagnetWindow.xib; sourceTree = "<group>"; };
+		0836CBED281E7EF700715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/GroupRules.xib; sourceTree = "<group>"; };
+		0836CBEE281E7EF700715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/InfoGeneralView.xib; sourceTree = "<group>"; };
+		0836CBEF281E7EF700715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/InfoActivityView.xib; sourceTree = "<group>"; };
+		0836CBF0281E7EF700715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/InfoOptionsView.xib; sourceTree = "<group>"; };
+		0836CBF1281E7EF700715E0C /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.xib; name = Base; path = Base.lproj/GlobalOptionsPopover.xib; sourceTree = "<group>"; };
```

Comment:
Actually, you should revert those last three:
- InfoActivityView.xib
- InfoOptionsView.xib
- GlobalOptionsPopover.xib

Maybe you did the base internationalization in an older commit and rebased your changes?